restClient/stp.rest

```diff
@@ -0,0 +1,517 @@
+,1018,18 bottles of beer on the wall,Option 1,0.8760680318,1962-05-13,1960-01-01 04:10:14,00:00:51,42,92
+,1019,19 bottles of beer on the wall,Option 1,0.7419619592,1962-02-11,1960-01-01 13:25:43,00:00:50,42,40
+,1020,20 bottles of beer on the wall,Option 1,0.1590175327,1962-09-22,1960-01-01 10:33:42,00:00:12,42,31
+,1021,21 bottles of beer on the wall,Option 1,0.4065937374,1960-03-24,1960-01-01 04:20:29,00:00:06,42,16
+,1022,22 bottles of beer on the wall,Option 1,0.4175353848,1960-09-08,1960-01-01 08:15:52,00:00:38,42,22
+,1023,23 bottles of beer on the wall,Option 1,0.0281551574,1962-02-08,1960-01-01 10:59:27,00:00:06,42,5
+,1024,24 bottles of beer on the wall,Option 1,0.3859465068,1961-07-26,1960-01-01 05:30:49,00:00:26,42,71
+,1017,17 bottles of beer on the wall,Option 1,0.4213511461,1960-09-19,1960-01-01 00:05:18,00:01:10,3,40
+,1025,25 bottles of beer on the wall,Option 1,0.1332434412,1960-12-01,1960-01-01 08:19:01,00:01:08,3,72
+,1026,26 bottles of beer on the wall,Option 1,0.8931343238,1962-07-06,1960-01-01 02:48:14,00:01:00,3,9
+,1027,27 bottles of beer on the wall,Option 1,0.4217192863,1960-02-16,1960-01-01 12:23:57,00:01:20,3,36
+,1028,28 bottles of beer on the wall,Option 1,0.3168482489,1961-09-15,1960-01-01 06:43:22,00:00:42,3,70
+,1029,29 bottles of beer on the wall,Option 1,0.6794722991,1962-09-06,1960-01-01 05:44:43,00:01:04,3,47
+,1030,30 bottles of beer on the wall,Option 1,0.2769889423,1962-05-26,1960-01-01 13:04:55,00:01:13,3,23
+,1031,31 bottles of beer on the wall,Option 1,0.7937810485,1960-08-01,1960-01-01 12:23:15,00:01:37,3,55
+,1032,32 bottles of beer on the wall,Option 1,0.7618695534,1961-11-22,1960-01-01 09:26:56,00:00:57,3,94
+,1033,33 bottles of beer on the wall,Option 1,0.9798893793,1960-05-21,1960-01-01 06:45:43,00:00:13,3,42
+,1034,34 bottles of beer on the wall,Option 1,0.131745664,1960-04-17,1960-01-01 11:14:20,00:01:39,3,27
+,1035,35 bottles of beer on the wall,Option 1,0.8229696834,1961-04-15,1960-01-01 04:07:12,00:00:50,3,24
+,1036,36 bottles of beer on the wall,Option 1,0.0351993381,1960-07-27,1960-01-01 12:51:57,00:00:50,3,63
+,1037,37 bottles of beer on the wall,Option 1,0.7755885626,1960-11-23,1960-01-01 12:49:10,00:00:59,3,21
+,1038,38 bottles of beer on the wall,Option 1,0.2894518693,1961-06-02,1960-01-01 05:01:18,00:00:27,3,95
+,1040,40 bottles of beer on the wall,Option 1,0.775269912,1962-09-23,1960-01-01 01:44:17,00:00:02,89,33
+,1041,41 bottles of beer on the wall,Option 1,0.7068485993,1961-05-13,1960-01-01 04:18:56,00:00:36,89,71
+,1042,42 bottles of beer on the wall,Option 1,0.845455137,1961-12-14,1960-01-01 11:08:23,00:00:35,89,63
+,1043,43 bottles of beer on the wall,Option 1,0.9986143364,1962-01-01,1960-01-01 09:51:26,00:01:21,89,43
+,1044,44 bottles of beer on the wall,Option 1,0.0831163596,1960-10-31,1960-01-01 03:34:26,00:00:41,89,92
+,1046,46 bottles of beer on the wall,Option 1,0.296541955,1961-07-21,1960-01-01 11:41:47,00:00:38,89,11
```

Comment:
not sure if it's necessary to have so much sample data?

Reply:
yea, let me reduce the sample data.

Reply:
updated